### PR TITLE
New MVC GSOC 2014

### PR DESCRIPTION
**Pull request by wilsonge**
https://groups.google.com/forum/#!topic/joomla-dev-cms/AH6piuw2IR0

http://gwilson.org/index.php/new-mvc

### CODE REVIEW

**Comment by Hackwar** [comment]
Please fix the codestyle issues that Travis is complaining about.

File: libraries/cms/component/dispatcher.php [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+		$controller = $this->getController($app);
+		$controller->options = $this->getTasks();
+
+		try
+		{
+			// Note we won't use the result because if the controller has executed successfully it still
+			// should have set a relevent error and redirect
+			$result = $controller->execute();
+		}
+		catch (Exception $e)
```
Comment (wilsonge) [inline libraries/cms/component/dispatcher.php]: I really have no clue what i was talking about here reviewing this. will remove all these

File: libraries/cms/controller/displayform.php [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+	 *
+	 * @return  boolean  True if the ID is in the edit list.
+	 *
+	 * @since   12.2
+	 */
+	protected function checkEditId($context, $id)
+	{
+		if ($id)
+		{
+			// Fix this check which is also a bug
```
Comment (wilsonge) [inline libraries/cms/controller/displayform.php]: This is from the code buddhima and elin wrote. I'd say it needs a fix

**Comment by wilsonge** [comment]
Fixed the obvious one. There will be another that travis complains about but that's some code commented out because of a bug so that will be fixed when the bug gets fixed

**Comment by roland-d** [comment]
@wilsonge Are we going anywhere with this PR?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/4137">issues.joomla.org/joomla-cms/4137</a>.</sub>

**Comment by wilsonge** [comment]
Not for now

**Pull request closed without merging**